test/Feature/EntryPointMissing.c

```diff
@@ -0,0 +1,46 @@
+// REQUIRES: posix-runtime
+// REQUIRES: uclibc
+// XFAIL: *
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=missing %t.bc 2>&1 | FileCheck -check-prefix=CHECK-MISSING %s
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=missing --libc=uclibc %t.bc 2>&1 | FileCheck -check-prefix=CHECK-MISSING-UCLIBC %s
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=missing %t.bc --posix-runtime 2>&1 | FileCheck -check-prefix=CHECK-MISSING-POSIX %s
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=missing %t.bc --libc=uclibc --posix-runtime 2>&1 | FileCheck -check-prefix=CHECK-MISSING-UCLIBC-POSIX %s
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=missing --libc=klee %t.bc 2>&1 | FileCheck -check-prefix=CHECK-MISSING-KLEELIBC %s
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=missing --libc=klee --posix-runtime %t.bc 2>&1 | FileCheck -check-prefix=CHECK-MISSING-KLEELIBC-POSIX %s
+
+// RUN: %clang -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out --entry-point=main --libc=uclibc --posix-runtime %t.bc 2>&1 | FileCheck -check-prefix=CHECK-MISSING-MAIN %s
+
+#include <stdio.h>
+
+int hello() {
+  printf("Hello World\n");
+  return 0;
+}
+
+// CHECK-MISSING: Entry function 'missing' not found in module
+// CHECK-MISSING-UCLIBC: Entry function 'missing' not found in module
+// CHECK-MISSING-POSIX: Entry function 'missing' not found in module
+// CHECK-MISSING-UCLIBC-POSIX: Entry function 'missing' not found in module
+// CHECK-MISSING-KLEELIBC: Entry function 'missing' not found in module
+// CHECK-MISSING-KLEELIBC-POSIX: Entry function 'missing' not found in module
+// CHECK-MISSING-MAIN: Entry function 'main' not found in module
```

Comment:
Why do we need more than two `CHECK` labels here? `CHECK-MISSING` and `CHECK-MISSING-MAIN` should be sufficient or do I miss sth.?

Reply:
It's now `CHECK-MISSING-MAIN` vs `CHECK-MAIN`.

Reply:
Thanks, @251, too much copy&paste!  Everything should be fixed now.